src/frontend/components/ViewersList/components/ViewersListTextButton/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Button, Text } from 'grommet';
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledTextButton = styled(Text)`
+  color: white;
+  font-family: 'Roboto-Bold';
+`;
+
+interface ViewersListTextButtonProps {
+  onClick: () => void;
+  text: string;
+}
+
+export const ViewersListTextButton = ({
+  onClick,
+  text,
+}: ViewersListTextButtonProps) => {
+  return (
+    <Button onClick={onClick} style={{ padding: '0px' }}>
```

Comment:
you are missing a11yTitle and title

Reply:
fix in [2679f8](https://github.com/openfun/marsha/pull/1443/commits/2679f83da20c408778dc1a90faf546eb37d51d4e)